### PR TITLE
Added future meetups and all that jazz

### PR DESCRIPTION
**Pull request by asgrim**
Yes.

### CODE REVIEW

**Comment by lboynton** [comment]
No.

File: data/meetups/2016-04-13.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+$eventbriteWidget .= '<span style="color:#888;"> for </span>';
+$eventbriteWidget .= '<a style="color:#888; text-decoration:none;" target="_blank" href="http://www.eventbrite.co.uk/event/' . $eid . '?ref=etckt">' . $etitle . '</a>';
+$eventbriteWidget .= '<span style="color:#888;"> powered by </span>';
+$eventbriteWidget .= '<a style="color:#888; text-decoration:none;" target="_blank" href="http://www.eventbrite.co.uk?ref=etckt">Eventbrite</a>';
+$eventbriteWidget .= '</div></div>';
+
+$meetup = new MeetupEntity();
+
+$abstract = <<<END
+This talk starts with a comparison of the different practices and tools we can use in the PHP ecosystem and is followed by a continuousphp demo. So that viewers can see a demo app tested and deployed in real time, with all the practices and tools explained before.
```
Comment (lboynton) [inline data/meetups/2016-04-13.php]: Comma after "continuousphp demo"?

File: data/meetups/2016-04-13.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+$meetup->setId(0)
+    ->setFromDate(new DateTime('2016-04-13 19:00'))
+    ->setToDate(new DateTime('2016-04-13 23:00'))
+    ->setRegistrationUrl("http://www.eventbrite.co.uk/event/{$eid}")
+    ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
+    ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
+    ->setTalkingPoints(array(
+    	new TalkEntity('Mike Rogers', 'MikeRogers0', '5 minute lightning talk'),
+    	new TalkEntity('Frederic Dewinne', 'fdewinne', 'The Continuous Talk', $abstract),
+        '&pound;20 Amazon.co.uk gift voucher prize draw, courtesy of Spectrum IT',
```
Comment (lboynton) [inline data/meetups/2016-04-13.php]: Should we put the Jetbrains thing here?

**Comment by asgrim** [comment]
@lboynton updated

File: data/meetups/2016-05-11.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+$meetup->setId(0)
+    ->setFromDate(new DateTime('2016-05-11 19:00'))
+    ->setToDate(new DateTime('2016-05-11 23:00'))
+    ->setRegistrationUrl("http://www.eventbrite.co.uk/event/{$eid}")
+    ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
+    ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
+    ->setTalkingPoints(array(
+    	'Lightning talk - tbc',
+    	new TalkEntity('Richard Harrison', 'phphants', 'Creating Unit Test Mocks - PHPUnit or Prophecy?', $abstract),
```
Comment (lboynton) [inline data/meetups/2016-05-11.php]: I guess Richard hasn't got a twitter account?

Comment (asgrim) [inline data/meetups/2016-05-11.php]: Not that I know of...

**Comment by lboynton** [comment]
Sorry I forgot about this in all my unread emails

**Pull request merged**